server/src/controllers/stuff.js

```diff
@@ -1,32 +1,35 @@
-const { getStuffData, addStuffItem,updateStuffItem, deleteStuffItem  } = require('../models/stuff');
- 
+const { getStuffData, addStuffItem, updateStuffItem, deleteStuffItem } = require('../models/stuff');
+
 exports.fetchStuffData = (req, res) => {
   getStuffData((err, result) => {
-    if (err) console.log(err) ;
+    if (err) console.log(err);
```

Comment:
`     if (err) console.log(err); `
i would suggest throwing an error and not just logging it 
actually i'm against logging it, please  
```suggestion
    if (err) throw err;
```